open-api/rest-catalog-open-api.yaml

```diff
@@ -2890,6 +2889,9 @@ components:
         snapshot-id:
           type: integer
           format: int64
+          description:
```

Comment:
this should probably also have a `deprecated: true` right before this line

Reply:
fixed in latest commit